packages/core/src/components/button/_common.scss

```diff
@@ -290,6 +290,13 @@ $button-intents: (
     box-shadow: none;
     background: $minimal-button-background-color-active;
     color: $pt-text-color;
+
+    &:disabled,
+    &:disabled:hover,
+    &.pt-disabled,
+    &.pt-disabled:hover {
+      color: $pt-text-color-disabled;
```

Comment:
this is not enough to fix the bug. background still disappears.

Reply:
hmm the previous commit did solve the issue, maybe I should revert it and leave as it was before

Reply:
previous commit was definitely closer to correct (look at the diff now--it's just about disabled styles), but you still had the problem of duplicated selectors (https://github.com/palantir/blueprint/pull/1738#discussion_r145777789)

Reply:
@llorca can you give us your opinion about what is the best solution here?

Reply:
is it because the disabled selectors come _after_ the active ones?

Reply:
ok, so in addition to this `color` rule, you need to redeclare `background: $minimal-button-background-color-active` from line 291, same in the dark theme. Otherwise it gets removed by line 306

Reply:
yes, you're right. Then we have to revert to the previous commit, we need the duplicated selectors